tools/topology/topology1/platform/intel/intel-generic-dmic.m4

```diff
@@ -46,7 +46,7 @@ dnl Unless explicitly specified, dmic period at 48k is 1ms
 ifdef(`DMIC_48k_PERIOD_US', `', `define(DMIC_48k_PERIOD_US, 1000)')
 
 ## Prolong period to 4ms for RTNR
-ifdef(`RTNR', `define(`INTEL_GENERIC_DMIC_PERIOD', 4000)', `define(`INTEL_GENERIC_DMIC_PERIOD', 1000)')
```

Comment:
This affects only the 16k pipeline. Just want to make sure you are aware of it.
The 48k one uses `DMIC_48k_PERIOD_US` which could be defined in the top topology.
I am wondering if we really need those `INTEL_GENERIC_DMIC_PERIOD` lines depending on `RTNR` being defined.

Reply:
Gah thanks for catching this, I must have made the mistake when doing my WIP patch cleanup